package.json

```
@@ -13,7 +13,7 @@
     "type": "git",
     "url": "git+https://github.com/1-Platform/one-platform.git"
   },
-  "version": "0.0.0",
+  "version": "0.1.0",
```

```suggestion
  "version": "0.1.0",
```
Any reason why the minor version was changed and not the patch @mudit94 ?

Or you can add each file with a commit and a patch version bump

---

@hybridx this needs a minor version change as it is a new feature integration in cypress